nginx/ngx_http_js_module.c

```diff
@@ -2972,6 +2972,33 @@ ngx_http_js_request_variables(njs_vm_t *vm, njs_object_prop_t *prop,
     name.len = val.length;
 
     if (setval == NULL) {
+        #if (NGX_PCRE)
```

Comment:
I just realized we do not need this #ifdef because njs module will not compile without nginx PCRE anyway.

Reply:
Ah, I haven't checked that. Removed the check then 👍